pages/common/vcpkg.md

```diff
@@ -0,0 +1,21 @@
+# vcpkg
+
+> C/C++ package manager.
+> Note: packages are not installed in the system. To use them, you need to tell your build system (e.g. cmake) to use vckg.
```

Comment:
```suggestion
> Note: packages are not installed in the system. To use them, you need to tell your build system (e.g. CMake) to use `vckg`.
```

Reply:
Why add quotes to vcpkg, but not say, CMake ? Is it because it is a command ? Because CMake will not use the vcpkg command, it will use the vcpkg environment

Reply:
Yes, see style [guide#descriptions](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions):
> ## Descriptions
> 
> - Avoid using the page title in the description (e.g. use `A sketching and painting program designed for digital artists` instead of `Krita is a sketching and painting program designed for digital artists`) unless the program name differs from the executable name (e.g. `rg` and Ripgrep).
> - Avoid mentioning that the program is used on the command-line (e.g. use `Ripgrep is a recursive line-oriented search tool` instead of `Ripgrep is a recursive line-oriented CLI search tool`).
> - Brand and project names can be capitalized in the description whenever applicable (e.g. use `A tool for interacting with a Git repository.` instead of ``A tool for interacting with a `git` repository.``).

Reply:
Don't worry, these backticks are far from being only for commands, they're used for al technical names that don't fall on the "brand" rule, like compression algorithms, for example (`zip`, `xz`, `7z`, ...). See [style guide#special-cases](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases):
> Technical wording on description lines should use the `backtick` syntax.
> Use backticks on the following:
> 
> - Paths, e.g. `package.json`, `/etc/package.json`.
> - Extensions, e.g. `.dll`.
> - Commands, e.g. `ls`.
> - Standard streams: `stdout`, `stdin`, `stderr`. **Do not** use the full names (e.g. standard output).